### ISSUE TITLE
docker image v5.6.5 doesn't have executables built

### ISSUE THREAD

**Issue by joovel** [Project-OSRM/osrm-backend#3907]
```
$ docker run -t -v $(pwd)/data/osrm:/data osrm/osrm-backend:v5.6.5 osrm-extract -p /opt/foot.lua /data/my-city.osm.pbf
docker: Error response from daemon: oci runtime error: container_linux.go:247: starting container process caused "exec: \"osrm-extract\": executable file not found in $PATH".
```

This does not happen on v5.6.4 or v5.6.3.

**Comment by danpat** [comment]
Ugh - I'll re-publish those images.  We turned on autobuilds on cloud.docker.com, but it picked up the wrong Dockerfile for this tag.

Hang tight, I'll re-publish the image.  You'll need to remove your local cached copy then re-pull.

**Comment by danpat** [comment]
@joovel I've pushed updated images.  Clean your local cache and give it another go.

This should be automatically fixed once the 5.7 release happens.

**Comment by joovel** [comment]
Confirmed. Thanks @danpat !
